src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,41 +1,50 @@
 package edu.kis.vh.nursery;
 
-import edu.kis.vh.nursery.list.IntLinkedList;
+import edu.kis.vh.nursery.container.IntArrayStack;
+import edu.kis.vh.nursery.container.IntLinkedList;
 
 public class DefaultCountingOutRhymer implements StackInterface {
 
-	IntLinkedList linkedList;
+	StackInterface stackInterface;

```

Comment:
"interface" suffix of the attribute name is wrong and does not show the meaning.